src/Cake.Common.Tests/Unit/Tools/VSTest/VSTestRunnerTests.cs

```diff
@@ -204,7 +204,7 @@ public void Should_Use_SettingsFile_If_Provided()
             var result = fixture.Run();
 
             // Then
-            Assert.Equal("\"/Working/Test1.dll\" /Settings:Local.RunSettings", result.Args);
+            Assert.Equal("\"/Working/Test1.dll\" /Settings:\"/Working/Local.RunSettings\"", result.Args);
```

Comment:
@jnm2 these are not integration tests, but rather unit tests, so there are no files on the file system.  If this is required, then we would mock those files using the IFileSystem interface.  However, I don't think that is required here, or did I miss something?  I would have thought we could just add another test similar to this one:

https://github.com/jnm2/cake/blob/d159d45b25120fdfa9a47112d05ddf44e258bcb8/src/Cake.Common.Tests/Unit/Tools/VSTest/VSTestRunnerTests.cs#L201

Where the created FilePath contains a space, and make sure that this is reflected correctly in the arguments that are created.  No?

No, I don't think that is required, but perhaps @devlead and @patriksvensson can chime in here as well.